constants/constants.js

```diff
@@ -245,6 +246,35 @@ export const formDataFields = {
   metadataField: 'entry.1027769131',
 };
 
+export const sitemapPaths = [
+        
+  '/sitemap.xml.gz',

```

Comment:
Please remove all patterns using .gz, .tar, .zip as we do not support compressed archives for sitemap